src/index.ts

```diff
@@ -20,7 +20,7 @@ import selectorParser from 'postcss-selector-parser';
 
 import { MinimalRenamer } from './minimal-renamer';
 
-export interface Options {
```

Comment:
I believe you can define `Options` as a member on the exported plugin, so TS users can write:

```typescript
import * as rename from 'postcss-rename';

let options: rename.Options;
```

I'm not 100% sure what the syntax is, but I think this might work:

```typescript
const plugin = postcss.plugin(/* ... */);

interface plugin.Options {
  // ...
}

export = plugin;
```

Reply:
Forgot about declaration merging! Done.